resource-managers/kubernetes/integration-tests/src/test/scala/org/apache/spark/deploy/kubernetes/integrationtest/KubernetesV2Suite.scala

```diff
@@ -87,6 +91,25 @@ private[spark] class KubernetesV2Suite extends SparkFunSuite with BeforeAndAfter
     runSparkAppAndVerifyCompletion(KubernetesSuite.CONTAINER_LOCAL_MAIN_APP_RESOURCE)
   }
 
+  test("Use remote resources without the resource staging server.") {
+    val assetServerUri = staticAssetServerLauncher.launchStaticAssetServer()
+    sparkConf.setJars(Seq(
+      s"$assetServerUri/${KubernetesSuite.EXAMPLES_JAR_FILE.getName}",
+      s"$assetServerUri/${KubernetesSuite.HELPER_JAR_FILE.getName}"
+    ))
+    runSparkAppAndVerifyCompletion(SparkLauncher.NO_RESOURCE)
```

Comment:
does this PR fix https://github.com/apache-spark-on-k8s/spark/issues/213 as a side effect?